net/librdkafka/Portfile

```diff
@@ -18,11 +18,42 @@ checksums           sha256 d6e553b1216b76de9e3a3c4b9c53fbecd87a5220aa5db2f3a8333
 
 patchfiles          CXX.patch
 
+configure.args-append --disable-zstd \
+                      --disable-lz4-ext \
+                      --disable-ssl \
+                      --disable-gssapi \
+                      --disable-syslog
+depends_lib           port:zlib
+                    
 if {[string match *clang* ${configure.cxx}]} {
     configure.ldflags-append -stdlib=${configure.cxx_stdlib}
 }
 
+variant zstd description {Enable support for zstd compression} {
+  depends_lib-append      port:zstd
+  configure.args-delete   --disable-zstd
+  configure.args-append   --enable-zstd
+}
+
+variant lz4 description {Enable external LZ4 library support} {
+  depends_lib-append      port:lz4
+  configure.args-delete   --disable-lz4-ext
+  configure.args-append   --enable-lz4-ext
+}
+
+variant ssl description {Enable SSL support} {
+  depends_lib-append      path:lib/libssl.dylib:openssl
+  configure.args-delete   --disable-ssl
+  configure.args-append   --enable-ssl
+}
+
+variant sasl description {Enable SASL GSSAPI support with Cyrus libsasl2} {
+  depends_lib-append port:cyrus-sasl2
+  configure.args-delete   --disable-gssapi
+  configure.args-append   --enable-gssapi
```

Comment:
Please use configure.args-replace instead of these two lines

Reply:
Thank you, fixed by ec0076ba2092f774228589e17ff37896859bdca0